README-docker-compose.md

```diff
@@ -129,10 +129,6 @@
       git checkout <master|develop|etc>
       docker buildx build --platform linux/arm64 -t osf:<branch>-arm64 .
       ```
-      - In `docker-compose.override.yml`, replace any `quay.io/centerforopenscience/osf:develop-arm64` with the locally-tagged image above:
-      ```yml
-      image: osf:<branch>-arm64
-      ```
```

Comment:
Because newer in-house Py3.12 images work on ARM, I might recommend removing the whole `- Building the Docker image` block, and instead saying something like 
```
  * _NOTE: The default `elasticsearch`, `elasticsearch6`, and `sharejs` containers are incompatible with ARM64._
  
  - To run `elasticsearch6` on ARM64 architecture:
  
    - Copy `docker-compose-dist-arm64.override.yml` into your `docker-compose.override.yml` file
  
  - Running containers with docker compose
  
    - In `webite/settings/local.py`, disable `SEARCH_ENGINE`
    ```python
    # SEARCH_ENGINE = 'elastic'
    SEARCH_ENGINE = None
    ```
    
    ## Application Runtime
    #[...]
```
Minor: newer versions of Docker/compose prefer `compose.override.yml`, but still support the old `docker-`-prefixed convention. Could include this information as well, but not strictly necessary